frontend/src/stories/components/Chat/ChatToggle.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
This story doesn't appear to be showing up on the storyboard, which suggests that there is some issue with it; storyboard is often not particularly good at showing error messages; the story just "doesn't show up" if its broken.

<img width="244" alt="image" src="https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-4/assets/1119017/8a24a70a-5180-491e-8ae7-32ea5632928a">

I''m willing to let it slide though since we can see the new button here: 

https://ucsb-cs156-f23.github.io/proj-happycows-f23-5pm-4/prs/28/storybook/?path=/story/pages-playpage--default